pkg/operator/ceph/cluster/controller.go

```diff
@@ -115,6 +116,8 @@ func newReconciler(mgr manager.Manager, ctx *clusterd.Context, clusterController
 		panic(err)
 	}
 
+	clusterController.recorder = k8sutil.NewEventReporter(mgr.GetEventRecorderFor("ClusterController"), 10, 5)
```

Comment:
Had commented on test code before. Can we change the rate-limiting to 1 every 10 minutes?

Reply:
@travisn and @leseb what do you think? IMO, we shoud minimize apiserver clutter wherever possible. Events are not logging. Each one is stored in the apiserver. They should only be an indication that something has gone wrong.

Reply:
Hmm, just had a thought: what if we have this sequence of events: failure(up to rate limiting), success, failure(event not thrown because of rate limiting).


Failure reconciles will be backed-off by controller-runtimes rate limiter, so maybe we should do something like a max of 1 per 2 minutes?

Reply:
Honestly, now I'm not understanding the purpose of the rate limiting. Why don't we report the event every time it happens, and if it repeats then we should increase the count. Increasing the count doesn't increase load significantly on the api server, right? If the reconcile fails again and again, it will naturally back off already. And the failure events should be less often since a healthy cluster will normally not have them.

Reply:
There is no count field, it's the client that groups them and shows the count.